### PR TITLE
Restoring egamma DQM sequence for allForPrompt (101X) 

### PR DESCRIPTION
**Pull request by fabozzi**
Fix to restore the egamma folder that is found to be missing in data certification of data re-reco

### CODE REVIEW

File: DQMOffline/Configuration/python/autoDQM.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
             'express':       ['@commonSiStripZeroBias+@muon+@hcal+@jetmet+@ecal',
                               'PostDQMOffline',
                               '@commonSiStripZeroBias+@muon+@hcal+@jetmet+@ecal'],
-            'allForPrompt':  ['@common+@muon+@hcal+@jetmet+@ecal',
+            'allForPrompt':  ['@common+@muon+@hcal+@jetmet+@ecal+@egamma',
                               'PostDQMOffline',
                               '@common+@muon+@hcal+@jetmet+@ecal'],
```
Comment (dmitrijus) [inline DQMOffline/Configuration/python/autoDQM.py]: needs to be added here too - this is for harvesting

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by fabozzi** [comment]
added also a backport for 94X

**Comment by fabiocos** [comment]
+1

**Pull request merged**